CONTRIBUTING.md

```diff
@@ -12,6 +12,26 @@ and file new ones, all feedback welcome!
 Before you can contribute, you will need to sign the [Contributor License
 Agreement](https://docs.linuxfoundation.org/docs/communitybridge/communitybridge-easycla/contributors).
 
+## Copyright Notices
+
+OpenTelemetry follows [CNCF recommendations](https://github.com/cncf/foundation/blob/master/copyright-notices.md)
+for copyright notices. We use "Copyright The OpenTelemetry Authors" notice form.
+
+According to CNCF recommendations if you are contributing a third-party code
+you will need to [retain the original copyright notice](https://github.com/cncf/foundation/blob/master/copyright-notices.md#dont-change-someone-elses-notice-without-their-permission).
+
+Any contributed third-party code must originally be Apache 2.0-Licensed or must
+carry a permisive software license that is compatible when combining with
+Apache 2.0 License (e.g. BSD or MIT license). 
```

Comment:
Do we have a list of known licenses for this case? As a developer, I really have no idea which licenses are compatible with the Apache 2.0 License.

Reply:
Pretty much what I listed: BSD, MIT and Apache 2.0 itself. See https://dwheeler.com/essays/floss-license-slide.html 
Arrows flowing in to Apache 2.0 are OK - resulting combined work is effectively Apache 2.0 licensed. Everything else on the diagram is prohibited since the resulting combined work no longer complies with Apache 2.0.

Reply:
Ok, the "e.g. BSD or MIT license" tricked me into thinking that there were others. Perhaps this would make it clearer:

```suggestion
Apache 2.0 License. At this moment, BSD and MIT are the only [OSI-approved licenses](https://opensource.org/licenses/alphabetical) known to be compatible.
```

If someone knows about any other compatible license, a PR can be opened to change this doc.

Reply:
Done.